service/syncer/syncer.go

```diff
@@ -67,6 +68,7 @@ func (s *Synchronizer) Sync(ctx context.Context, user *core.User) (*core.Batch,
 		if err := recover(); err != nil {
 			logger = logger.WithField("error", err)
 			logger.Errorln("syncer: unexpected panic")
+			debug.PrintStack()
```

Comment:
Maybe use the same logger to output the stack trace too, just to make sure the stack trace ends up in the same place as the rest of the logs.
```suggestion
			logger.Errorf("syncer: unexpected panic\n%s\n", debug.Stack())
```

Reply:
👍  updated now :)